Remove SDL_GL_CONTEXT_EGL #6657 (#6683)
libsdl-org/SDL: docs/README-migration.md

Programs which have access to shaders can implement more robust versions of those functions using custom shader code rendered as a post-process effect.

> SDL_GL_CONTEXT_EGL has been removed.

Reviewer: This should include what the new method is that replaces it.

Reviewer: This should also be added to the WhatsNew.txt file.

Programs which have access to shaders can implement more robust versions of those functions using custom shader code rendered as a post-process effect.

> Removed 'SDL_GL_CONTEXT_EGL' from OpenGL configuration attributes (eg SDL_video.h, SDL_GLattr)

Reviewer: ```suggestion
Removed 'SDL_GL_CONTEXT_EGL' from OpenGL configuration attributes
```

> You can get the previous behavior with 'SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_ES);'

Reviewer: ```suggestion
You can instead use 'SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_ES);'
```
